scene/gui/graph_node.cpp

```diff
@@ -458,6 +461,27 @@ void GraphNode::_shape() {
 	title_buf->add_string(title, font, font_size, opentype_features, (language != "") ? language : TranslationServer::get_singleton()->get_tool_locale());
 }
 
+#ifdef TOOLS_ENABLED
+void GraphNode::_edit_set_position(const Point2 &p_position) {
+	GraphEdit *graph = Object::cast_to<GraphEdit>(get_parent());
+	if (graph) {
+		Point2 offset = (p_position + graph->get_scroll_ofs()) * graph->get_zoom();
+		set_position_offset(offset);
+		set_position(p_position);
```

Comment:
`set_position()` should be outside this condition, otherwise the node can't be moved if not inside GraphEdit.